plugin_helpers/utils.py

```diff
@@ -48,3 +51,14 @@ def unique(seq):
 def rreplace(s, old, new, occurrence):
   li = s.rsplit(old, occurrence)
   return new.join(li)
+
+def quote(s):
+  if "windows" in platform.system().lower():
+    return _quote_windows_string(s)
+  else:
+    return shlex.quote(s)
+
+def _quote_windows_string(s):
+  if not s:
+    return "\"\""
+  return "\""+s.replace("\"", "\\\"")+"\""
```

Comment:
* Single quotes should make this more readable (`"\""` becomes `'"'`).
* Add spaces around `+`.